SwiftUIDZ/View/ClimateView.swift

```diff
@@ -0,0 +1,385 @@
+                    HStack {
+                        Text(Constants.descriptionText)
+                            .font(Font.system(size: 15, weight: .semibold))
+                            .foregroundColor(.gray)
+                        Spacer()
+                    }
+                }
+                .frame(width: 200)
+                Spacer()
+                Button {
+                    isTemperatureShown.toggle()
+                    climateViewModel.updateTemperature()
+                } label: {
+                    Image(Constants.settingText)
+                        .font(Font.system(size: 30, weight: .semibold))
+                        .foregroundColor(.gray)
+                        .background(
+                            Circle()
+                                .fill(
+                                    isTemperatureShown ? Color(Constants.topGradientColorName) :
+                                        Color(Constants.backgroundColorName)
+                                )
+                                .neumorphismSelectedStyle()
+                                .frame(width: 62, height: 62)
+                        )
+                        .neumorphismUnSelectedStyle()
+                }
+            }
+        }
+        .frame(width: UIScreen.main.bounds.width - 80)
+    }
+
+    private var footerSheetView: some View {
+        ZStack {
+            VStack {
+                HStack {
+                    Image(Constants.offText)
+                    Spacer()
+                    Stepper("\(Int(climateViewModel.stepperValue))\(Constants.degreeText)") {
+                        guard climateViewModel.stepperValue < Constants.maximumDegreeNumber else { return }
+                        climateViewModel.stepperValue += 1.0
+                        climateViewModel.updateTemperature()
+                    } onDecrement: {
+                        guard climateViewModel.stepperValue > Constants.minimumDegreeNumber else { return }
+                        climateViewModel.stepperValue -= 1.0
+                        climateViewModel.updateTemperature()
+                    }
+                    .frame(width: 150)
+                    Spacer()
+                    Image(Constants.unionText)
+                }
+                HStack {
+                    Text(Constants.onText)
+                    Spacer()
+                    ColorPicker(Constants.startText, selection: $climateViewModel.startColorValue)
+                        .frame(width: 85)
+                    Spacer()
+                    ColorPicker(Constants.finishText, selection: $climateViewModel.finishColorValue)
+                        .frame(width: 85)
+                    Spacer()
+                    Text(Constants.ventText)
+                }
+            }
+        }
+        .frame(width: UIScreen.main.bounds.width - 60)
+    }
+
+    // MARK: - Private Methods
+
+    private func climateSettingView(index: Int) -> some View {
```

Comment:
```suggestion
    private func makeClimateSettingView(index: Int) -> some View {
```
make всегда в начале МЕТОДА